templates/test/ci/patches/control-plane-kubeadm-boostrap-ci-version.yaml

```diff
@@ -14,54 +14,56 @@
       # and finally uninstalls the packages.
       # For the release packages it tests all versions in the support skew.
       LINE_SEPARATOR="*************************************************"
-      echo "$$LINE_SEPARATOR"
+      echo "$${LINE_SEPARATOR}"
       CI_VERSION=${CI_VERSION}
+
+      # Note: We assume if kubectl has the right version, everything else has as well
```

Comment:
This seems like a reasonable early exit. Was this change from upstream? Just curious.